pages/security/two-factor-authentication.mdx

```diff
@@ -5,6 +5,35 @@ import {Callout} from "nextra-theme-docs";
 
 Craftable Pro uses the [Laravel Fortify](https://laravel.com/docs/fortify) package to implement the backend part of the two-factor authentication.
 
+## Config
+
+CraftablePro overrides the default Fortify config with necessary value changes. However, if you have the `fortify.php` config
+already published or plan to publish it into your project, you will have to make a few changes to make it work with CraftablePro's
+two-factor authentication feature.
+
+```
+'guard' => 'craftable-pro',
+'views' => false,
+'features' => [
+        //Features::registration(),
+        //Features::resetPasswords(),
+        //Features::emailVerification(),
+        //Features::updateProfileInformation(),
+        //Features::updatePasswords(),
+        Features::twoFactorAuthentication([
+            'confirm' => true,
+            'confirmPassword' => true,
+            // 'window' => 0,
+        ]),
+    ],
+```
+Same as they are in `vendor/brackets/craftable-pro/config/fortify.php`
+
+Not all the features have to be disabled, but the Fortify registration, verification and password handling have not yet
+been tested within CraftablePro, as it comes with its own implementations.
+
+If the `'views'` would be set to `true`, Fortify would register its own routes for 2FA and there would be route naming conflicts when caching routes.
```

Comment:
Is there a way to fix this on our side, so there would no be conflicts?

Shouldn't we prefix **all** our routes with `craftable-pro.` prefix?

Reply:
@palypster we made a frontend for the Fortify routes, so if we do not want to rewrite Fortify code we have to keep some routes without craftable-pro, the same way they are named within the Fortify package. Maybe there is some easy way to use a differently named route (such as through macros) - this is how Laravel's ResetPassword feature works. It is possible to use any route url through macro, without touching the Laravel vendor code.

However I don't think it is a pressing issue, since there is no harm in setting `views => false` as we do not want to mix craftable and fortify 2FA frontends together. And we have not tested it with `views => true`, we might find more problems if we did. So it would be a bigger task, not for 5 minutes.